spec/models/exercise_spec.rb

```diff
@@ -9,12 +9,14 @@
       expect(exercise).to be_valid
     end
 
-    it 'persists a valid model object when given strings for title, general description, user id and a global boolean' do
+    it 'is valid given a string for a title and persists general description, user id and a global boolean' do
```

Comment:
Sorry to be really picky here, but it still does not persist anything and shouldn't.

Reply:
Actually, thank you for this pickiness.

This teaches me that I misunderstood and misapplied the word "persist" here.

> In computer science, persistence refers to the characteristic of state that outlives the process that created it. This is achieved in practice by storing the state as data in computer data storage. Programs have to transfer data to and from storage devices and have to provide mappings from the native programming-language data structures to the storage device data structures.
--- https://en.wikipedia.org/wiki/Persistence_%28computer_science%29

I was using "persist" to mean "saved in an object" rather than "saved from the object INTO a database." That's wrong. 

So thanks!